stratum/stratum.cpp

```diff
@@ -137,6 +137,7 @@ YAAMP_ALGO g_algos[] =
 	{"lyra2", lyra2re_hash, 0x80, 0, 0},
 	{"lyra2v2", lyra2v2_hash, 0x100, 0, 0},
 	{"lyra2z", lyra2z_hash, 0x100, 0, 0},
+	{"lbk3", lbk3_hash, 0x100, 0, 0},
```

Comment:
an algo of this speed (in GHs) should not have a factor of 256

Reply:
What factor should it be then?

Reply:
yes, 1 is fine, but miners should have the same factor

Reply:
ok... so